root/frontend/src/styles/PreviewCard.scss

```diff
@@ -0,0 +1,63 @@
+.previewCard {
+	width: 200px;
+	background: #4e525b;
+	max-height: 250px;
+	border-radius: 15px;
+}
+
+.previewCardHeader {
+	display: flex;
+	align-items: center;
+	height: 50px;
+	width: 100%;
+}
+
+.profileImage {
+	border-radius: 100%;
+	width: 30px;
+	height: 30px;
+	margin: 0px 10px 0px 10px;
+}
+.previewCardTitle {
+	color: white;
+	height: 33px;
```

Comment:
Is there any particular reason why this is `33px`? It seems that the profile image is `30px` height-wise so it seems a little odd to me.

Reply:
The `align-items: center` isn't exactly perfect, so I offset the difference in the height to fix it.